apps/api-gateway/src/main.ts

```
@@ -1,19 +1,27 @@
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
   const config = new DocumentBuilder()
     .setTitle('BisonTrade')
     .setDescription('The BisonTrade API description')
     .setVersion('1.0')
+    .addBearerAuth()
     .build();
   const documentFactory = () => SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('api', app, documentFactory);
+  app.use('/api-json', (_: any, res: { json: (arg0: Document) => void }) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)